.github/workflows/publish.yml

```
@@ -0,0 +1,35 @@
+name: Build
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Get the Yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(corepack yarn config get cacheFolder)"
+
+      - uses: actions/cache@v3
+        with:
+          path: ${{steps.yarn-cache-dir-path.outputs.dir}}
+          key: ${{runner.os}}-yarn-${{hashFiles('**/yarn.lock')}}
+          restore-keys: |
+            ${{runner.os}}-yarn-
+
+      - name: Publish the package
+        run:
+          corepack yarn install --immutable
```

We want to publish the built files, right?
```suggestion
          corepack yarn install --immutable
          corepack yarn build
```

---

It's automatically called as part of the [`prepack` script](https://github.com/nodejs/corepack/blob/main/package.json#L60)